Multiprotocol/KF606_ccnrf.ino

```diff
@@ -107,6 +120,15 @@ static void __attribute__((unused)) KF606_initialize_txid()
 		hopping_frequency[0]=68;
 		hopping_frequency[1]=71;
 	#endif
+	if(sub_protocol == KF606_ZCZ50)
```

Comment:
Change 123..131 by:
```
	if(sub_protocol == KF606_ZCZ50)
	{
		rx_tx_addr[1] = rx_tx_addr[0];
		rx_tx_addr[0]=0xAA;
	}
#ifdef FORCE_ZCZ50_ORIGINAL_ID
		rx_tx_addr[0]=0xAA;
		rx_tx_addr[1]=0x67;
		rx_tx_addr[2]=0x64;
		rx_tx_addr[3]=0x01;
		hopping_frequency[0]=48;
		hopping_frequency[1]=51;
#endif
```

Reply:
This should enable multiple IDs to work if you do not define FORCE_ZCZ50_ORIGINAL_ID